app/build.gradle.kts

```diff
@@ -16,14 +16,13 @@ plugins {
 apply(from = "update_instances.gradle.kts")
 
 android {
-    buildToolsVersion = "34.0.0"
```

Comment:
This was to align the buildToolsVersion with the one from the CI image

This prevents the CI from downloading that each time and shaves of like 3m of the pipeline.

Was more important back then. And the version could be different by know.

Reply:
Didn't seem to make a difference in the CI:

- [PR with buildToolsVersion specified](https://woodpecker.join-lemmy.org/repos/137/pipeline/2801/5#L9)
- [This PR without it](https://woodpecker.join-lemmy.org/repos/137/pipeline/2795/5#L9)

Reply:
Ye seems to not be working for a while now.